src/tests.rs

```diff
@@ -1130,8 +1132,10 @@ pub mod serverless_executor_test {
             block_number: Some(block_number),
             address: H160::from_str(JOBS_CONTRACT_ADDR).unwrap(),
             topics: vec![
-                keccak256("JobCreated(uint256,address,bytes32,bytes,uint256,address[])").into(),
+                keccak256("JobCreated(uint256,uint8,address,bytes32,bytes,uint256,address[])")
+                    .into(),
                 H256::from_uint(&job_id),
+                H256::from_uint(&EXECUTION_ENV_ID.into()),
                 H256::from(H160::random()),
```

Comment:
can you add a case where job is created for unsupported environment?

Reply:
Sure will do. Btw Do we need to send slashing txn for different env job requests or do we just drop them overall?

Reply:
We should just send the slashing txn for current env only.